src/main/java/com/blackducksoftware/integration/hub/model/view/VulnerabilityWithRemediationView.java

```diff
@@ -29,25 +29,19 @@
 
 public class VulnerabilityWithRemediationView extends HubView {
     public String vulnerabilityName;
-
     public String description;
-
+    public Date vulnerabilityPublishedDate;
     public Date vulnerabilityUpdatedDate;
-
     public double baseScore;
-
     public double impactSubscore;
-
     public double exploitabilitySubscore;
-
     public String source;
-
     public String severity;
-
     public String remediationStatus;
-
+    public String cweId;
```

Comment:
CWE - Common Weakness Enumeration (trademark of MITRE Corporation). BlackDuck is adding those ids in all vulnerabilities